### ISSUE TITLE
Config > QEMU tab > "Advanced: Bypass configuration..." doesn't do anything

### ISSUE THREAD

**Issue by bartgrantham** [utmapp/UTM#2348]
**Describe the issue**
The "Advanced: Bypass configuration..." checkbox in the QEMU tab under VM settings doesn't do anything.  The options stay grayed out.

**Configuration**
* UTM Version: 2.0.22
* OS Version: macOS 11.2.2
* Apple Silicon

**Comment by conath** [comment]
I can confirm the toggle works as expected. However, because the first few lines of QEMU arguments remain the same and can't be changed, it *looks* like nothing happens. 

You need to scroll down the list to see that the generated arguments have been removed and you can now insert you own ones. Another tip: press the TAB ⇥ key after entering your last QEMU argument before you click `Save` to make sure it gets saved.

<img width="1012" alt="scrollDown" src="https://user-images.githubusercontent.com/12073163/109420267-c549b580-79d1-11eb-8627-52bbf4451bc8.png">
<img width="1012" alt="addCustomBottom" src="https://user-images.githubusercontent.com/12073163/109420270-c7ac0f80-79d1-11eb-89a7-a93e4c07b49d.png">

**Comment by bartgrantham** [comment]
Yup.  I see it now.  I'll change the issue to be more UI-oriented.

**Comment by 8thHalfHour** [comment]
So, to be clear (because I was really uncertain how to apply my own QEMU arguments on this page — it wasn’t super clear to me that “bypass configuration. . . “ was going to allow me to add new arguments), if we want to add arguments, all of the others there get ignored? Or are the extra arguments added to what exists? Additionally, are all of the arguments necessary for UTM to run or are the also customizable? If so, how?

Thank you!

**Comment by osy** [comment]
If you check “bypass configurations” it does exactly that and deletes all the auto generated options except the ones that gets the UI to work. If you just add arguments at the bottom they will be appended to all the other ones. You can always use the export button to check the complete command it generates.

**Comment by 8thHalfHour** [comment]
Oh, okay. So , if I understand correctly, those commands that are greyed must remain, and then what we add with “bypass” will also get sent, yes? Is there a way, then, to keep all of the other commands and just add to them?

**Comment by osy** [comment]
Yes scroll to the bottom and add it. But I would rather you tell me what’s missing so I can try working it into the gui.
